packages/contracts-bedrock/scripts/merkle-trie-fuzzer/go.mod

```diff
@@ -0,0 +1,11 @@
+module oplabs.co/merkle-trie-fuzzer
```

Comment:
Wondering where this should live, we would want the module name to be `github.com/ethereum-optimism/optimism/contracts-bedrock/<path-to-directory>`

A few options:
- `contracts-bedrock/fuzz` or `contracts-bedrock/fuzzer`
- `contracts-bedrock/scripts/fuzzer`

I'm leaning towards it being a top level package named `fuzz` or `fuzzer`

Any opinions @trianglesphere @protolambda @maurelian @smartcontracts?

Reply:
Agree with "top level" (in `packages/contracts-bedrock`).

Maybe call it `go-fuzz`, as I expect there to be other fuzzing approaches as well

Reply:
Got it- moved to top level as `go-fuzzers`